setup.py

```diff
@@ -16,7 +16,7 @@
 from setuptools import setup, find_packages
 
 
-VERSION = "0.22.1"
+VERSION = "0.22.2"
```

Comment:
I noticed that this was already bumped on `develop`, so shouldn't it be here too?

Reply:
it should be 0.23.0 on develop and 0.22.2 here, sure